Polygon_mesh_processing/test/Polygon_mesh_processing/connected_component_surface_mesh.cpp

```diff
@@ -43,56 +44,63 @@ struct Constraint : public boost::put_get_helper<bool,Constraint<G> >{
   double bound;
 };
 
+template <typename G, typename GT>
+struct Face_descriptor_area_functor
+{
+  typedef typename boost::graph_traits<G>::face_descriptor     face_descriptor;
+
+  Face_descriptor_area_functor(const G& g, const GT& gt) : g(g), gt(gt) { }
 
-int main(int argc, char* argv[])
+  double operator()(const face_descriptor f) const
```

Comment:
this `double`  should be `GT::FT`, right?

Reply:
`FT` is double since `Simple_cartesian<double>` is used everywhere, but I cleaned the whole file nevertheless.